railties/lib/rails/templates/rails/mailers/email.html.erb

```diff
@@ -131,15 +133,24 @@
   function refreshBody() {
     var part_select = document.querySelector('select#part');
     var locale_select = document.querySelector('select#locale');
-    var part_param = part_select.options[part_select.selectedIndex].value;
-    var locale_param = locale_select.options[locale_select.selectedIndex].value;
     var iframe = document.getElementsByName('messageBody')[0];
-    iframe.contentWindow.location = '?' + part_param + '&' + locale_param;
+    var part_param = part_select ?
+      part_select.options[part_select.selectedIndex].value :
+      document.querySelector('#mime_type').dataset.mime-type;
```

Comment:
`dataset.mime-type` must be `dataset.mimeType`.

Reply:
🙇🏻